api/tests/certification/enrolment/unit/domain/models/Center_test.js

```diff
@@ -22,33 +77,27 @@ describe('Unit | Certification | Session | Domain | Models | Center', function (
       expect(center).to.be.an.instanceOf(Center);
       expect(center.hasBillingMode).is.true;
     });
+  });
 
-    it('should return true when center is a complementary certification pilot', function () {
+  context('#isComplementaryAlonePilot', function () {
+    it('should return false when center is not a complementary alone pilot', function () {
```

Comment:
deplacer `should return true when center is a complementary certification pilot` ici!

Reply:
Done